doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -31,9 +57,9 @@ The FIWARE NGSI (Next Generation Service Interface) API defines
 (whether to separate the two interfaces is currently under discussion).
 * a set of typical **roles** played by NGSI-compliant components
 
-### Context data modelling and exchange
+## Terminology
```

Comment:
The problem is that the current approach is "flatering" sections that was original thought with an structure (one section for NGSI9 terms and another section for NGSI10 terms, each one wich subsections). A possible solution would be:

```
## Terminology for context data modelling and exchange
(subsections using ###)
## Terminology for context availability information
(subsectiosn using ###)
```
but I don't like it too much...

@tobiasjacobs feedback is valuable here, as he was the one proposing these sections.